modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_kazimi/kazimi_test.i

```diff
@@ -0,0 +1,90 @@
+#liquid sodium properties at @773 [K]
+rho = 762.90
+vel = 0.1
+k = 64.217
+mu = 2.358e-4
+cp = 1264.6
+T = 773
+T_wall = 774
+D_h = 0.1
+PoD = 1.1
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 1
+  allow_renumbering = false
+[]
+
+[Variables]
```

Comment:
I made a comment a while back on this. You don't need to solve a dummy diffusion problem. See that comment.

Reply:
Yes, sorry about that, I just saw your comments today, already changed all the files